src/applications/gi/components/vet-tec/VetTecSearchForm.jsx

```diff
@@ -115,6 +132,22 @@ class VetTecSearchForm extends React.Component {
     );
   };
 
+  renderCautionaryWarningsFilter = () => {
```

Comment:
can you break this out into its own component like how https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/components/search/OnlineClassesFilter.jsx is ?

Reply:
Sure